org.mixedrealitytoolkit.spatialmanipulation/BoundsControl/BoundsHandleInteractable.cs

```diff
@@ -102,6 +104,14 @@ protected override void Awake()
             HideOnStartup();
         }
 
+        /// <summary>
+        /// A Unity event function that is called every frame, if this object is enabled.
+        /// </summary>
+        protected virtual void Update()
+        {
+            hasUpdated = true;
```

Comment:
Can we validate the initialization state in LateUpdate() without adding an additional flag?